apps/jumpcloud/src/inngest/functions/users/delete-users.ts

```diff
@@ -0,0 +1,30 @@
+import { NonRetriableError } from 'inngest';
+import { db } from '@/database/client';
+import { Organisation } from '@/database/schema';
+import { inngest } from '@/inngest/client';
+import { deleteUsers } from '@/connectors/users';
+import { decrypt } from '@/common/crypto';
+
+export const deleteSourceUsers = inngest.createFunction(
+  { id: 'delete-users' },
+  { event: 'jumpcloud/users.delete.requested' },
+  async ({ event }) => {
+    const { userId } = event.data;
+
+    const [organisation] = await db
+      .select({
+        apiKey: Organisation.apiKey,
+      })
+      .from(Organisation);
```

Comment:
I miss that in the previous PR review. But, we should select the org using id retrieved from event data.
```suggestion
      .from(Organisation)
      .where(eq(Organisation.id, organisationId));
```